Remotion/Validation/Core/Validators/ScalePrecisionValidator.cs

```diff
@@ -40,8 +51,15 @@ protected override bool IsValid (PropertyValidatorContext context)
       if (!(context.PropertyValue is decimal propertyValue))
         return true;
 
-      var scale = GetScale (propertyValue);
-      var precision = GetPrecision (propertyValue);
+      var decimalAsString = propertyValue.ToString (CultureInfo.InvariantCulture);
```

Comment:
hmmm...use format-modifier, too? (e.g. "N"?)

Reply:
Format modifiers without an explicit precision specifier seem to output decimal with a scale of 2 as default, which would not be beneficial here.